localisation/00_zuwc_united_localisation.csv

```diff
@@ -2025,6 +2025,16 @@ book_of_medivh;Book of Medivh;Livre de Medivh;;;;;;;;;;;;x
 book_of_medivh_desc;The spellbook written by Medivh.;;;;;;;;;;;;;x
 top_hat;Fancy Top Hat;;;;;;;;;;;;;x
 top_hat_desc;With this top hat, you will definitely attract attention to yourself!;;;;;;;;;;;;;x
+robes_of_arugal;Robes of Arugal;;;;;;;;;;;;;x
+robes_of_arugal_desc;Robes worn by Arugal, former royal archmage of Gilneas and member of the Kirin Tor.;;;;;;;;;;;;;x
+belt_of_arugal;Belt of Arugal;;;;;;;;;;;;;x
+belt_of_arugal_desc;Belt worn by Arugal, former royal archmage of Gilneas and member of the Kirin Tor.;;;;;;;;;;;;;x
+remedy_of_arugal;Remedy of Arugal;;;;;;;;;;;;;x
+remedy_of_arugal_desc;"How Arugal gained acceptance within the Kirin Tor is beyond me. His spell-casting knowledge seemed as transparent as a blown glass bauble. Arugal used enchanted items to reinforce his weak magic." Improves Learning at the expense of one's reputation.;;;;;;;;;;;;;x
+meteor_shard;Meteor Shard;;;;;;;;;;;;;x
+meteor_shard_desc;Dagger wielded by Arugal, former royal archmage of Gilneas and member of the Kirin Tor. Melee attacks have a chance to blast the target with Fire magic, making this weapon deceptively powerful.;;;;;;;;;;;;;x
+book_of_ur;The Book of Ur;;;;;;;;;;;;;x
+book_of_ur_desc;Written by Ur of Dalaran, this book contains a detailed account on the origins of the Worgen. "It is my hope that no Dalaran wizard seeks out the Worgen, even if done in light conscience. For no pact may be struck, no secrets may be learned, no good can come from these beasts. They are best left to their world. For if found in ours and not destroyed, our peril will be dire..." ;;;;;;;;;;;;;x
```

Comment:
Put new lines in the end of file, otherwise there will be conflicts on other branches